lib/verbose_migrations.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'logger'
+
+module VerboseMigrations
+  module MigrationExtension
+    cattr_accessor :verbosity, default: nil
+
+    def verbose? = verbosity.present? && ActiveRecord::Base.logger.present?
+    def verbose!(level: Logger::DEBUG)
```

Comment:
Add `verbose_logger: ActiveRecord::Base.logger` so the logger is configurable, since we may want to e.g. make `Keygen.logger` verbose at times.